.github/workflows/release.yml

```diff
@@ -23,6 +26,18 @@ jobs:
       with:
         generate_release_notes: true
         make_latest: true
+    - name: Log in to the Container registry
+      uses: docker/login-action@v3
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - uses: docker/setup-qemu-action@v3
+    - uses: docker/setup-buildx-action@v3
+    - uses: docker/build-push-action@v6
+      with:
+        push: true
+        tags: ${{ env.REGISTRY }}/${{ github.repository_owner }}/sprocket:${{ github.ref_name }}
```

Comment:
Just checking, Is `github.ref_name` going to be the `v*` tag?

Reply:
Yes, I believe so. I modeled it after: https://github.com/stjude-rust-labs/sprocket/blob/3760b07e894f31f15fd623a05ec6685c29105683/.github/workflows/release.yml#L70. I'm open to whether the `v` prefix should be included, but since it is used in the release and in the artifacts, I opted to include it here.

Reply:
LGTM